Source/OxyPlot/Graphics/Snapping.cs

```diff
@@ -0,0 +1,39 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="Snapping.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Snapping data.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot
+{
+    /// <summary>
+    /// Snapping data.
+    /// </summary>
+    public class Snapping
```

Comment:
would it be better to add simple properties `IsSnappingEnabled` and `SnappingPrecision` to the axis? I am concerned about these snapping options being a mutable reference type. Also, I don't think it works well with the WPF axis "wrappers" - no change notifications if you change an `IsEnabled` property